tests/browser-tests/Login.browser.ts

```diff
@@ -1,6 +1,8 @@
 import { DashboardPage, FilesPage, LoginPage, AuthorDetailsPage, DetailsPage, NavigationPane } from '../page-objects';
 
-fixture`Getting Started`.page`http://localhost:9000`;
+const BASE_URL = process.env.BASE_URL || 'localhost:9000';
```

Comment:
Just spit balling on the developer experience, but does it make sense to move this into a local module? Then you could have 
```JavaScript
import { BASE_URL } from './path/to/my-module'
```
Then the remember-to-always-define-the-constant turns into just another import. Might be easier to document, too.

Will defer to @Cooryd and @will-byrne who have the experience in this codebase as to whether this is a good idea or not.

Reply:
As this is blocking me right now, I'll merge but am happy to make another PR to implement something like this.

Reply:
I like this suggestion

Reply:
PR implementing this at https://github.com/libero/reviewer-client/pull/247